packages/terra-action-header/src/ActionHeader.jsx

```diff
@@ -69,7 +69,7 @@ const contextTypes = {
   /* eslint-disable consistent-return */
   intl: (context) => {
     if (context.intl === undefined) {
-      return new Error('Please add locale prop to Base component to load translations');
+      return new Error('Component requires Localization, and must be wrapped in terra-base');
```

Comment:
Would it make more sense to state that the component is internationalized and must be wrapped in base rather than saying it requires localization?

Reply:
Spoke with @JakeLaCombe regarding error text changes, he agrees with the changes suggested.